numalogic/registry/artifact.py

```diff
@@ -38,14 +38,18 @@ def __init__(self, uri: str):
 
     @abstractmethod
     def load(
-        self, skeys: Sequence[str], dkeys: Sequence[str], latest: bool = True, version: str = None
+        self,
+        skeys: Sequence[str],
+        dkeys: Sequence[str],
+        production: bool = True,
+        version: str = None,
     ) -> ArtifactData:
         """
         Loads the desired artifact from mlflow registry and returns it.
         Args:
             skeys: static key fields as list/tuple of strings
             dkeys: dynamic key fields as list/tuple of strings
-            latest: boolean field to determine if latest version is desired or not
+            production: boolean field to determine if latest version is desired or not
```

Comment:
nitpick: when we need to explain `production` with `latest`, maybe we should just use the `latest` term to avoid extra mental mapping.

Further more, `production` seems indicating a use case while `latest` indicates the desired behavior in the function.

Reply:
That seems logical.